docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,183 @@
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+</div>
+<div class="tab-pane fade" id="setup-windows" markdown="1">
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+</div>
+</div>
+
+## Install Crossplane
+Install the alpha version of Crossplane, which is suitable for community use and testing:
+<ul class="nav nav-tabs">
+<li class="active"><a href="#install-tab-helm3" data-toggle="tab">Helm 3</a></li>
+<li><a href="#install-tab-helm2" data-toggle="tab">Helm 2</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="install-tab-helm3" markdown="1">
+```
+kubectl create namespace crossplane-system
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+</div>
+<div class="tab-pane fade" id="install-tab-helm2" markdown="1">
+```
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+</div>
+</div>
+
+## Install Crossplane CLI
+The Crossplane CLI adds a set of `kubectl crossplane` commands to simplify common tasks:
+```
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane-cli/master/bootstrap.sh | bash
+```
+
+## Select Provider
+In order for Crossplane to be able to manage resources in a specific cloud
+provider, you will need to install a provider and configure it for Crossplane to use.
+<ul class="nav nav-tabs">
+<li class="active"><a href="#provider-tab-gcp" data-toggle="tab">GCP</a></li>
+<li><a href="#provider-tab-aws" data-toggle="tab">AWS</a></li>
+<li><a href="#provider-tab-azure" data-toggle="tab">Azure</a></li>
+<li><a href="#provider-tab-alibaba" data-toggle="tab">Alibaba</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="provider-tab-gcp" markdown="1">
+### Install GCP Provider
+```
+PACKAGE=crossplane/provider-gcp:v0.9.0
+NAME=provider-gcp
+kubectl crossplane package install --cluster --namespace crossplane-system ${PACKAGE} ${NAME}
+```
+
+### Create gcp-account-creds.json
+The following `curl | bash` script that creates a service account and grants
+access to infrastructure services. Alternatively you use the [Configure] docs
+to manually generate account credentials.
+```
+# replace this with your own gcp project id
+GCP_PROJECT_ID=TODO_REPLACE
+curl -s https://raw.githubusercontent.com/crossplane/stack-gcp-sample/master/hack/gcp-credentials.sh | PROJECT_ID=${GCP_PROJECT_ID} bash -
+```
```

Comment:
I would prefer not to try to script this for folks. I think at best it's a scary proposition to run someone's script with your cloud provider credentials, and at worst it will lead to a poor UX when we inevitably forget to update this script when new resources that need new IAM roles are added.

I'm wondering whether we really have to automate and/or teach the process of creating a new project and granting IAM roles to a service account. I can't find any examples of similar infra-automation projects including those in their documentation. If we left this out the total process would be something like two commands: create the service account, and save it as a secret.

Reply:
done